---
target_selection: blue
mission_targets:
  blue:
    gate_target_frame: gate_blue_arrow_link
    red_buoy_direction: ccw
    slalom_direction: left_side
    bin_target_frame: bin/blue_link
    torpedo_target_frame: torpedo_map_link
    octagon_target_frame: octagon_link
  red:
    gate_target_frame: gate_red_arrow_link
    red_buoy_direction: cw
    slalom_direction: right_side
    bin_target_frame: bin/red_link
    torpedo_target_frame: torpedo_map_link
    octagon_target_frame: octagon_link
